src/components/givePost.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+    <div class="container" >
+    <div id="main-heading" >
+      <h1 > {{title}} </h1>
+    </div>
+    <div>
+      <p> {{body}} </p>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+    props:["title", "body"]
+}
+</script>
+
+<style >
+.container{
+  width:90%;
+  margin-left: 5%;
+  margin-right:5%;
+  margin-top:50px;
+}
+#main-heading{
```

Comment:
@ChoudharyHamza `class` is proffered for styles unless needed

Reply:
ok used class from this point on and removed the id's used on previous elements